getting-started/codecs.md

```diff
@@ -42,26 +41,35 @@ Tremor's internal data representation is JSON-like. The supported value types ar
 
 ### Codecs
 
-Tremor neither requires nor validates schemas and works with schemaless or unstructured
-data. Validation can be asserted with the tremor-script language. `Onramps`, `Offramps`
-and other components of tremor may however require or expect conformance with schemas.
+Tremor neither requires nor validates schemas and works with schemaless or unstructured data. Validation can be asserted with the tremor-script language. `Onramps`, `Offramps`, and other components of Tremor may, however, require or expect conformance with schemas.
 
-For specific components their documentation should be consulted for correct usage.
+For specific components, their documentation should be consulted for correct usage.
 
 Tremor supports the encoding and decoding of the following formats:
 
+<<<<<<< HEAD
 * [json](/docs/Artefacts/codecs#json)
 * [msgpack](/docs/Artefacts/codecs#msgpack)
 * [influx](/docs/Artefacts/codecs#influx)
 * [binflux](/docs/Artefacts/codecs#binflux) - (binary representation of the influx wire protocol)
 * [statsd](/docs/Artefacts/codecs#statsd)
 * [yaml](/docs/Artefacts/codecs#yaml)
 * [string](/docs/Artefacts/codecs#string) - any valid UTF-8 string sequence
+=======
+* [json](https://docs.tremor.rs/artefacts/codecs#json)
+* [msgpack](https://docs.tremor.rs/artefacts/codecs#msgpack)
+* [influx](https://docs.tremor.rs/artefacts/codecs#influx)
+* [binflux](https://docs.tremor.rs/artefacts/codecs#binflux)- (binary representation of the influx wire protocol).
+* [statsd](https://docs.tremor.rs/artefacts/codecs#statsd)
+* [yaml](https://docs.tremor.rs/artefacts/codecs#yaml)
+* [string](https://docs.tremor.rs/artefacts/codecs#string)- any valid UTF-8 string sequence.
+>>>>>>> 22a80a7 (Make homepage feature headers links)
 
 <h3 class="section-head" id="h-concept"><a href="#h-codecs"></a>Pre- and Postprocessors</h3>
 
 Tremor supports the following preprocessing transformations in `Onramp` configurations:
 
+<<<<<<< HEAD
```

Comment:
Another merge artifact, I think we want to keep the relative path instead of the full one, since all pages are now the same